src/Mod/Draft/Draft.py

```diff
@@ -170,6 +170,81 @@
 getMovableChildren = draftutils.utils.get_movable_children
 get_movable_children = draftutils.utils.get_movable_children
 
+#---------------------------------------------------------------------------
+# Draft objects
+#---------------------------------------------------------------------------
+
+
+
+#---------------------------------------------------------------------------
+# Draft annotation objects
+#---------------------------------------------------------------------------
+
+from draftobjects.dimension import make_dimension, make_angular_dimension
+from draftobjects.dimension import LinearDimension, AngularDimension
+
+makeDimension = make_dimension
+makeAngularDimension = make_angular_dimension
+_Dimension = LinearDimension
+_AngularDimension = AngularDimension
+
+if gui:
+    from draftviewproviders.view_dimension import ViewProviderLinearDimension
+    from draftviewproviders.view_dimension import ViewProviderAngularDimension
+    _ViewProviderDimension = ViewProviderLinearDimension
+
+_ViewProviderAngularDimension = ViewProviderAngularDimension
```

Comment:
In Travis the gui doesn't exist. So this view provider also has to be inside the `if gui:` so it is skipped.

Reply:
good catch!